public/app/features/dashboard/components/PanelEditor/OverrideEditor.tsx

```diff
@@ -114,8 +103,13 @@ export const OverrideEditor: React.FC<OverrideEditorProps> = ({
     });
 
   const renderOverrideTitle = (isExpanded: boolean) => {
-    const overriddenProperites = override.properties.map(p => registry.get(p.id).name).join(', ');
+    const overriddenProperites = override.properties
+      .map(p => registry.getIfExists(p.id)?.name)
+      .filter(name => !!name)
+      .join(', ');
+
```

Comment:
why do we do this (map to just name)? and then map over the properties (the names) and call registry.getIfExists again? why not keep this be the registry items and map over them later. 

So would be jus to filter out the undefined / null ones.

Reply:
same question :)

Reply:
Not really sure I am following. Can you explain a bit more how you think I could change it?

Reply:
Sorry I misread the diff wrong. My comment does not make sense. 

But after reading the code more, It feels like we can get properties and filter out null items once, now it's in two places.  (in renderOverrideTitle and in the body), and two calls to registry.get, think we can simplify the code a bit that way.  

```ts
const configProps =  override.properties
      .map(p => registry.getIfExists(p.id))
      .filter(p => p !== null)
```

Reply:
Super, I will fix that tomorrow.